content/docs/concepts/security/index.md

```diff
@@ -10,197 +10,230 @@ aliases:
+Sidecar and perimeter proxies for implementing secure communication between clients and servers,
+Pilot for distributing [authentication policies](/docs/concepts/security/#authentication-policies)
+and [secure naming information](/docs/concepts/security/#secure-naming) to the proxies,
+and Mixer for managing authorization and auditing.
+
+{{< image width="80%" ratio="56.25%"
+    link="./architecture.svg"
+    alt="Istio architecture."
     caption="Istio Security Architecture"
     >}}
 
-As illustrated in the diagram, Istio uses secret volume mounts to deliver
-keys and certificates from Citadel to Kubernetes containers. For services
-running on VMs or bare-metal machines, we introduce a node agent, which is a
-process running on each VM or bare-metal machine. The node agent generates the
-private key and the CSR (Certificate Signing Request) locally, sends the CSR to
-Citadel for signing, and delivers the generated certificate together with the
-private key to Envoy.
+### Identity provision
+
+Istio PKI, built on top of Istio Citadel,
+is responsible for securely provisioning strong workload identities to to every workload.
+The identities are in SPIFFE format, carried by X.509 certificates.
+The PKI also automates the key & certificate rotation and revocation at scale.
+
+### Service and user authentication
 
-## Mutual TLS authentication
+Istio provides two different types of authentication:
 
-### Identity
+- **Service-to-service Authentication** enables data-in-transit encryption and strong channel-level authentication using
+  [mutual TLS](https://en.wikipedia.org/wiki/Mutual_authentication).
 
-Istio uses [Kubernetes service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/)
-to identify who runs the service:
+- **User Authentication** enables request-level authentication with JSON Web Token (JWT) validation
+  and a streamlined developer experience for [Auth0](https://auth0.com/), [Firebase Auth](https://firebase.google.com/docs/auth/),
+  [Google Auth](https://developers.google.com/identity/protocols/OpenIDConnect), and custom auth.
 
-- A service account in Istio has the format:
-  `spiffe://\<_domain_\>/ns/\<_namespace_>/sa/\<_serviceaccount_\>`.
+In both cases the authentication policies are stored in the Istio config store via a custom Kubernetes API,
+and are kept up to date (along with keys where appropriate) for each proxy by Pilot.
+Istio also supports authentication in permissive mode to understand how a policy change would affect your security posture
+before it becomes effective.
 
-    - Replace `_domain_` with `_cluster.local_`. We will support customization
-      of domain in the near future.
-    - Replace `_namespace_` with the namespace of the Kubernetes service
-      account.
-    - Replace `_serviceaccount_` with the Kubernetes service account name.
+For more details,  please read the [authentication](/docs/concepts/security/#authentication) section.
```

Comment:
remove extra space before "please"

Reply:
It is considered bad practice to use "please" on technical content. I suggest removing it.